fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -303,5 +303,67 @@ def parsed_info
     def automated_scheme_selection?
       !!ENV["AUTOMATED_SCHEME_SELECTION"]
     end
+
+    def backticks(command, print: true, timeout: false, retries: 0)
```

Comment:
Do you mean the `Helper.backticks`? If yes, this is a private version that adds timeout and retry mainly because in [this PR](https://github.com/fastlane/fastlane/pull/4492) it was suggested that having this locally to the one spot it is currently needed would be better.

I think the approach there is better because it adds this functionality to whomever needs it but I decided to open a simplified version to discuss it.